src/main/java/az/edu/turing/module03/lesson02.sql

```diff
@@ -0,0 +1,88 @@
+--docker run --name homework_postgre -e POSTGRES_PASSWORD=root -p 5445:80 -d postgres
+--docker start [enter id]
+--docker exec -it [id] psql -U postgres
+--CREATE DATABASE test;
+--\c test
+CREATE TABLE IF NOT EXISTS users(
+id                  bigserial,
+created_at          timestamptz(3)                          DEFAULT CURRENT_TIMESTAMP,
+updated_at          timestamptz(3)                          DEFAULT CURRENT_TIMESTAMP,
+updated_by          bigint,
+email               varchar(50) UNIQUE NOT NULL,
+name                varchar(20) NOT NULL,
+surname             varchar(30) NOT NULL,
+display_name        varchar(40) NOT NULL,
+gender              varchar(10) NOT NULL,
+age                 integer CHECK(age > 0),
+is_active           boolean                                 DEFAULT TRUE,
+PRIMARY KEY(ID),
+CONSTRAINT fk_user FOREIGN KEY(updated_by) REFERENCES users(id));
+
+CREATE TABLE IF NOT EXISTS posts(
+id                  bigserial,
+created_at          timestamptz(3)                          DEFAULT CURRENT_TIMESTAMP,
+updated_at          timestamptz(3)                          DEFAULT CURRENT_TIMESTAMP,
+like_count          integer CHECK(like_count >= 0)          DEFAULT 0,
+comment_count       integer CHECK(comment_count >= 0)       DEFAULT 0,
```

Comment:
I think you should have relation with `comments` table to get count not like that